src/services/__tests__/tourettes.test.js

```diff
@@ -18,31 +18,49 @@ describe('tourettes', () => {
   describe('sayTourettes', () => {
     describe('when input text is empty', () => {
       const inputText = '';
+      const minAmountOfReplies = 3;
 
       it('replies with at least three items from the profanities list', () => {
         const reply = sayTourettes(inputText);
+        const amountOfReplies = reply.split(' ').length;
         expect(reply).toNotBe(undefined);
-        expect(reply.split(' ').length).toBeGreaterThan(3);
+        expect(amountOfReplies).toBeGreaterThanOrEqualTo(minAmountOfReplies);
+      });
+    });
+
+    describe('when input text is integer', () => {
+      const amountOfProfanities = 25;
```

Comment:
Obscuring the value is just adding cognitive load, or trying too hard, it should be easy to just add a value to a variable, it should make sense and not make you think :)